featurebyte/worker/task_executor.py

```diff
@@ -130,6 +130,8 @@ def _setup_worker_config(self) -> None:
             "default_profile: worker\n\n",
             encoding="utf-8",
         )
+        # Reload newly written configuration
+        Configurations(force=True)
```

Comment:
I realise one user facing difference compared to previous behaviour is that within a python interpreter session, any changes to the config file will not have effect until `Configurations(force=True)` is run, whereas before the changes will be picked up immediately. Is that right @bushwhackr? 

The previous behaviour seems a bit more convenient, wondering if it's worth preserving (something like only caching `_config_file_path`, but always re-parse the config file when instantiating `Configurations`).

Reply:
Okay if we always reparse I think there is no need to put the force/reload param

Reply:
reverted reparsing code, due to lack of ability to run 2 servers at the same time.